crates/ruff/src/rules/flake8_logging_format/rules.rs

```diff
@@ -10,7 +10,7 @@ use crate::rules::flake8_logging_format::violations::{
     LoggingRedundantExcInfo, LoggingStringConcat, LoggingStringFormat, LoggingWarn,
 };
 
-enum LoggingLevel {
+pub enum LoggingLevel {
```

Comment:
Perhaps this should be moved to a different file now

Reply:
Let's move this to a new `ast/logging.rs`? (`ast` is a bit of a misnomer, but I think it's right to be alongside moduiles like `ast/whitespace.rs`.)